tests/test_ui.py

```
@@ -91,51 +79,41 @@ def test_ui_plain_non_tty_skips_prompts(monkeypatch):
-def test_ui_input_interactive_uses_gum(monkeypatch):
-    monkeypatch.setattr(ui_module.shutil, "which", lambda _cmd: "/usr/bin/fake")
+def test_ui_input_interactive_uses_questionary(monkeypatch):
     calls = []
 
-    def fake_run(cmd, **_kwargs):
-        calls.append(cmd)
-        if cmd[:2] == ["gum", "input"]:
-            return SimpleNamespace(stdout="value\n", stderr="", returncode=0)
-        return SimpleNamespace(stdout="", stderr="", returncode=0)
+    class DummyText:
+        def __init__(self, prompt: str, default: str):
```

Ruff (0.15.9)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)